src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -2437,6 +2431,96 @@ def update_policy_setdefinition(cmd, policy_set_definition_name, definitions=Non
     return policy_client.policy_set_definitions.create_or_update(policy_set_definition_name, parameters)
 
 
+def create_policy_exemption(cmd, name, policy_assignment=None, exemption_category=None,
+                            policy_definition_reference_ids=None, expires_on=None,
+                            display_name=None, description=None, resource_group_name=None, scope=None,
+                            metadata=None):
+    if policy_assignment is None:
+        raise RequiredArgumentMissingError('--policy_assignment is required')
+    if exemption_category is None:
+        raise RequiredArgumentMissingError('--exemption_category is required')
+
+    policy_client = _resource_policy_client_factory(cmd.cli_ctx)
```

Comment:
could we use cf_policy_exemptions as you defined in command type?

Reply:
cf_policy_exemptions is not defined in this file. And I needs subscription_id from the policy_client object. It doesn't help remove this line.

Reply:
How policy_client.config.subscription_id get the sub id value then?

Reply:
It is the same as other policy commands to get the subscription Id from the context (current selected subscription)

Reply:
@robga Actually, the `subscription_id` of ` policy_client.config` comes from `cli_ctx`, why not get `subscription_id` directly from `cli_ctx`?
Such as `cmd.cli_ctx.data['subscription_id']`

Reply:
OK, I will try it.